com.sap.cloud.lm.sl.cf.process/src/test/java/com/sap/cloud/lm/sl/cf/process/mock/MockDelegateExecution.java

```diff
@@ -327,4 +284,198 @@ public String getSuperExecutionId() {
         return null;
     }
 
+    @Override
+    public Object getTransientVariable(String arg0) {
+        // TODO Auto-generated method stub
```

Comment:
Can we remove the TODOs from this class? We obviously don't intend to implement them in any way.